src/components/editMeasure/testCases/components/testCaseLanding/qdm/CreateCodeCoverageNavTabs.tsx

```diff
@@ -190,88 +190,97 @@ export default function CreateCodeCoverageNavTabs(props: NavTabProps) {
           />
         </Tabs>
         <div style={{ margin: "6px 0 0 auto", display: "flex", gap: "10px" }}>
-          <Button
-            variant="danger-primary"
-            disabled={!canEdit || measure?.testCases?.length === 0}
-            onClick={onDeleteAllTestCases}
-            data-testid="delete-all-test-cases-button"
-          >
-            <KeyboardArrowRightIcon
-              style={{ margin: "0 5px 0 -2px" }}
-              fontSize="small"
-            />
-            Delete All
-          </Button>
-
-          <Button
-            onClick={() => {
-              if (onImportTestCases) {
-                onImportTestCases();
-              }
-            }}
-            variant="outline"
-            disabled={!canEdit}
-            data-testid="show-import-test-cases-button"
-          >
-            <FileUploadIcon
-              style={{ margin: "0 5px 0 -2px" }}
-              fontSize="small"
-            />
-            Import from Bonnie
-          </Button>
+          {!featureFlags.TestCaseListActionCenter && (
+            //Remove button when feature flag is removed
+            <Button
+              variant="danger-primary"
+              disabled={!canEdit || measure?.testCases?.length === 0}
+              onClick={onDeleteAllTestCases}
+              data-testid="delete-all-test-cases-button"
+            >
+              <KeyboardArrowRightIcon
+                style={{ margin: "0 5px 0 -2px" }}
+                fontSize="small"
+              />
+              Delete All
+            </Button>
+          )}
+          {((featureFlags.TestCaseListActionCenter && canEdit) ||
+            !featureFlags.TestCaseListActionCenter) && (
```

Comment:
looks like regardless of flag we wan to display these buttons. so, this could be just can edit check?
```suggestion
          { canEdit && (
```

Reply:
AC specifically says otherwise
> The Import buttons and the New Case button should not appear on the screen if the user does own the measure or it's not shared with them OR if it's not in a draft state